### PR TITLE
Implement destination section

### PR DESCRIPTION
**Pull request by JennyDoJD**
**Design**
Mobile
image
Desktop
image

**Result**
Mobile
image
image
image

Tablet
image
image
image

Desktop
image

### CODE REVIEW

File: practice/src/css/sections/destination.css [hunk 1]
```diff
@@ -0,0 +1,143 @@
...
+
+.destination-rate-number {
+  color: var(--light-text-color);
+}
+
+.destination-description {
+  padding-bottom: 50px;
+}
+
+.destination-button {
```
Comment (huyencao-agilityio) [inline practice/src/css/sections/destination.css]: Why do you split style for .destination-button 

Please help douple check 
```
.destination-button {
  text-align: center;
} 
```

File: practice/src/css/sections/destination.css [hunk 2]
```diff
@@ -0,0 +1,143 @@
...
+  padding-top: 50px;
+}
+
+.destination-card-item-image {
```
Comment (huyencao-agilityio) [inline practice/src/css/sections/destination.css]: Why do you split style for destination-card-item-image

```
.destination-card-item-image {
  width: 100%;
  height: auto;
  margin-bottom: 20px;
  border-radius: 24px;
}
```

File: practice/src/css/sections/destination.css [hunk 3]
```diff
@@ -0,0 +1,143 @@
...
+  }
+
+  .destination-heading {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+  }
+
+  .destination-card-item {
+    display: block !important;
```
Comment (huyencao-agilityio) [inline practice/src/css/sections/destination.css]: Avoid use !important to override

File: practice/src/css/sections/destination.css [hunk 4]
```diff
@@ -0,0 +1,143 @@
...
+  font-weight: var(--font-weight-semibold);
+  font-size: var(--font-size-medium);
+  line-height: var(--secondary-line-height);
+}
+
+.destination-unit {
+  line-height: var(--tertiary-line-height);
+}
+
+.destination-card-item {
```
Comment (huyencao-agilityio) [inline practice/src/css/sections/destination.css:96]: Merge style for .destination-card-item

File: practice/src/index.html [hunk 5]
```diff
@@ -70,6 +70,213 @@ <h5 class="service-topic-title">Easy trip booking</h5>
...
+            </div>
+            <div class="destination-rate">
+              <img src="../src/assets/images/icon-star.svg" class="icon-star">
+              <span class="destination-rate-number">4.9</span>
+            </div>
+            <div class="destination-address">
+              <span class="destination-type-trip">Hiking</span>
+              <h4 class="destination-title">Parau Mountain</h4>
+              <div class="destination-substact">
+                <img src="../src/assets/images/icon-subtract-destination.svg" class="icon-substact">
```
Comment (huyencao-agilityio) [inline practice/src/index.html]: Should be add alt attribute for the image. Douple check all images tag

File: practice/src/index.html [hunk 6]
```diff
@@ -70,6 +70,213 @@ <h5 class="service-topic-title">Easy trip booking</h5>
...
+    <section class="destination-section">
+      <!-- Destination section -->
+      <div class="container wrapper-destination">
+        <hgroup class="text destination-heading">
+          <h3 class="secondary-title">Explore Natural Destinations</h3>
+          <p class="description destination-description">There are many choices of mountain climbing and other tourist attractions from various regions in Indonesia</p>
+        </hgroup>
+        <div class="text destination-card">
+          <div class="destination-card-item">
+            <div class="destination-card-image">
```
Comment (huyencao-agilityio) [inline practice/src/index.html]: Should use `figure` instead `div`

**Comment by huyencao-agilityio** [comment]
@JennyDoJD Conflict

File: practice/src/css/sections/destination.css (hunk 2 above)
Comment (JennyDoJD) [reply practice/src/css/sections/destination.css]: Because I added the raw image, not the design image on figma

**Pull request merged**